lib/addons/p5.dom.js

```diff
@@ -961,6 +1041,7 @@
           }
           this.elt.setAttribute('width', aW * this._pInst._pixelDensity);
           this.elt.setAttribute('height', aH * this._pInst._pixelDensity);
+          this.elt.setAttribute('style', 'width:' + aW + 'px; height:' + aH + 'px;');
           this.elt.setAttribute('style', 'width:' + aW + 'px; height:' + aH + 'px');
```

Comment:
I'm not sure why that line is there... I think there was a merge issue at one point and I overlooked that line, sorry!